tests/security_features_test.go

```diff
@@ -262,22 +245,11 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 			}
 			caps := *container.SecurityContext.Capabilities
 			if !checks.HasFeature(virtconfig.Root) {
-				Expect(caps.Add).To(HaveLen(1), fmt.Sprintf("Found capabilities %s, expected NET_BIND_SERVICE", caps.Add))
-				Expect(caps.Add).To(ContainElement(k8sv1.Capability("NET_BIND_SERVICE")))
-			} else {
-				Expect(caps.Add).To(HaveLen(2), fmt.Sprintf("Found capabilities %s, expected NET_BIND_SERVICE and SYS_NICE", caps.Add))
-				Expect(caps.Add).To(ContainElements(k8sv1.Capability("NET_BIND_SERVICE"), k8sv1.Capability("SYS_NICE")))
-			}
-
-			By("Checking virt-launcher Pod's compute container has precisely the documented extra capabilities")
-			for _, capa := range caps.Add {
-				Expect(isLauncherCapabilityValid(capa)).To(BeTrue(), "Expected compute container of virt_launcher to be granted only specific capabilities")
-			}
-
-			if !checks.HasFeature(virtconfig.Root) {
+				Expect(caps.Add).To(ConsistOf(k8sv1.Capability("NET_BIND_SERVICE")))
 				By("Checking virt-launcher Pod's compute container has precisely the documented dropped capabilities")
-				Expect(caps.Drop).To(HaveLen(1))
-				Expect(caps.Drop[0]).To(Equal(k8sv1.Capability("ALL")), "Expected compute container of virt_launcher to drop all caps")
+				Expect(caps.Drop).To(ConsistOf(k8sv1.Capability("ALL")), "Expected compute container of virt_launcher to drop all caps")
+			} else {
+				Expect(caps.Add).To(ConsistOf(k8sv1.Capability("NET_BIND_SERVICE"), k8sv1.Capability("SYS_NICE")))
```

Comment:
Here probably worth  to have assert for `Drop` as well.

Reply:
Done